src/computer.cpp

```diff
@@ -1689,8 +1689,12 @@ void computer::mark_refugee_center()
 SEARCHING FOR NEAREST REFUGEE CENTER, PLEASE WAIT ... " ) );
 
     const mission_type_id &mission_type = mission_type_id( "MISSION_REACH_REFUGEE_CENTER" );
-    const std::vector<mission *> missions = g->u.get_active_missions();
     tripoint mission_target;
+    // Check completed missions too, so people can't repeatedly get the mission.
+    const std::vector<mission *> completed_missions = g->u.get_completed_missions();
+    std::vector<mission *> missions = g->u.get_active_missions();
+    missions.reserve( completed_missions.size() );
```

Comment:
If you want this `reserve` call to do the sensible thing it needs to be `missions.reserve( missions.size() + completed_missions.size() );`.  But in fact it's unnecessary.  The range overload of `vector::insert` is already smart enough to only do a single allocation.

Reply:
That's what I get for looking to stack overflow for answers. Thanks, I'll remove this line.